### ISSUE TITLE
Need to abstract sparkl_cli functions for import and re-use

### ISSUE THREAD

**Issue by jacobythwaites** [sparkl/cli#23]
For automated tests and other devops functions, it would be useful to have the sparkl_cli functions exposed in a form where they can be imported and used in other scripts.

Something like:
```
from sparkl_cli import (
    cmd_connect as connect,
    cmd_login as login,
    cmd_put as upload,
    cmd_call as invoke,
    cmd_vars as vars)

connect("http://localhost:8000")
login("test@sparkl.com", "postoffice", register=True)
upload("MyMix.xml", "Scratch")
vars.set(n=13)
invoke("Scratch/MyMix/Solicit")
```

**Comment by jacobythwaites** [comment]
Library usage mirrors command line as follows:
```
from sparkl_cli.main import sparkl

result1 = sparkl("connect", url="http://localhost:8000")
result2 = sparkl("login", user="foo@bar.com", password="something")
result3 = sparkl("list", folder="/Scratch")
```
etc

**Comment by jacobythwaites** [comment]
Fixed by #25
